src/cli/restore/groups.go

```diff
@@ -91,6 +92,10 @@ func restoreGroupsCmd(cmd *cobra.Command, args []string) error {
 	sel := utils.IncludeGroupsRestoreDataSelectors(ctx, opts)
```

Comment:
Quick note: I didn't add the exclude change inside `IncludeGroupsRestoreDataSelectors` because details cmd also uses it. I don't see a big impact here, since this is a short term change.